Procfile

```diff
@@ -1 +1,3 @@
-bundle exec thin start -R config.ru -e $RACK_ENV -p $PORT
\ No newline at end of file
+deploy: bundle exec rake seed:admins
```

Comment:
This is a question that I don't have the answer to: would placing this command in the procfile run it every time the application is deployed?